<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addAlarmDialog</class>
 <widget class="QWidget" name="addAlarmDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>283</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QTimeEdit" name="timeEdit">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="displayFormat">
      <string>hh:mm</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="okayButton">
     <property name="text">
      <string>Okay</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QCalendarWidget" name="calendarWidget"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
